compiler/front/main.nim

```diff
@@ -223,6 +223,9 @@ proc commandView(graph: ModuleGraph) =
 const
   PrintRopeCacheStats = false
 
+when PrintRopeCacheStats:
+    import utils/ropes
```

Comment:
```suggestion
  import utils/ropes
```
fixing the indentation, this should be compiler code base standardizes on two space indent.